aiida_quantumespresso/cli/utils/options.py

```diff
@@ -4,15 +4,46 @@
 
 from aiida.cmdline.params import types
 from aiida.cmdline.params.options import OverridableOption
+from aiida.cmdline.utils import decorators
+from aiida.common import exceptions
 
 from . import validate
 
-STRUCTURE = OverridableOption(
-    '-s', '--structure', type=types.DataParamType(sub_classes=('aiida.data:structure',)), help='StructureData node.'
-)
+
+class PseudoFamilyType(types.GroupParamType):
```

Comment:
Not sure, in this case I made a custom type in order to print a custom message. This will help people that pass an old `Group` or `UpfFamily` that they used to use as pseudopotential family, but will now no longer work. At least this way they can quickly install a new group that *will* work with the new interface. I am not sure whether all other plugins need this.

Reply:
I figured the warning message could also be used more generally, e.g. in case someone tries to use a upf family for running a Siesta `Calcjob`. We can change the link to the `aiida-pseudo` README and add the supported types there.

There also seems to be the `PseudoPotentialFamilyParam` class in the `aiida-pseudo` package, which has a similar purpose? I suppose its usage in the `show` command could also benefit from the warning in case someone tries to apply it to an old `Group` or `UpfFamily`.

Nothing critical though, just a thought.